Gems/AtomTressFX/Code/Passes/HairParentPass.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <Atom/RPI.Public/Pass/ParentPass.h>
+#include <AtomCore/std/containers/array_view.h>
+#include <AzCore/std/containers/fixed_vector.h>
+
+namespace AZ
+{
+    namespace Render
+    {
+        //! HairParentPass owns and manages ShadowmapPasses.
+        class HairParentPass final
```

Comment:
Nothing you added to the HairParentPass seems to be used (I can't see any code that uses m_slotName, m_updateChildren, or GetPipelineViewTags). Can you just use the regular ParentPass?

Reply:
After adding comments about the reason for the class, I'll keep it for now unless we decide to remove it for now.  The reason to keep is to allow ability to change pipeline on the fly when choosing which render method to use and if to enable collisions.